### PR TITLE
generate-text-file 0.0.1

### PR DESCRIPTION
**Pull request by nicolas-fricke**
This step allows users to create a generic text file they can use further down in the process. This is useful for e.g. creating a _What's New_ message for the Play Store deploy step based on the Git commit messages.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [X] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [X] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [X] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [X] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/generate-text-file/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+website: https://github.com/nicolas-fricke/bitrise-step-generate-text-file
+source_code_url: https://github.com/nicolas-fricke/bitrise-step-generate-text-file
+support_url: https://github.com/nicolas-fricke/bitrise-step-generate-text-file/issues
+published_at: 2017-02-10T13:14:04.386721078+01:00
+source:
+  git: https://github.com/nicolas-fricke/bitrise-step-generate-text-file.git
+  commit: 1dd81d6acaf45388a446f85427cb89abdfb64726
+host_os_tags:
+- osx-10.10
+project_type_tags:
```
Comment (viktorbenei) [inline steps/generate-text-file/0.0.1/step.yml]: You should remove all the project type tags, as this step should work with any kind of project (right?)

File: steps/generate-text-file/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- android
+- xamarin
+type_tags:
+- script
+- file
+- versioning
```
Comment (viktorbenei) [inline steps/generate-text-file/0.0.1/step.yml]: please remove all the non relevant tags, leave only "file" and "utils" (IMO)

File: steps/generate-text-file/0.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    is_required: true
+    title: File Name
+- file_content: null
+  opts:
+    is_expand: true
+    is_required: true
+    summary: Add content for the text file here. You can also use all ENV variables.
+    title: Content of the text file
+outputs:
+- GENERATED_TEXT_FILE_PATH: null
```
Comment (viktorbenei) [inline steps/generate-text-file/0.0.1/step.yml]: Not an issue if you leave this here, but as you have a "file name" input, this output is redundant IMO

Comment (nicolas-fricke) [inline steps/generate-text-file/0.0.1/step.yml]: For my own workflow it'd be really nice to have it as a public env variable afterwards. That's why I added it here.

**Comment by nicolas-fricke** [comment]
Made the changes as requested and opened a PR for a new version ☝️ 

Closing this PR.

**Pull request closed without merging**